conda_smithy/templates/build_steps.sh.tmpl

```diff
@@ -76,11 +76,11 @@ ulimit -n 1024
 # make the build number clobber
 make_build_number "${FEEDSTOCK_ROOT}" "${RECIPE_ROOT}" "${CONFIG_FILE}"
 
-{% if test == "native" -%}
+{% if test == "native" and conda_build_tool != "rattler-build" -%}
 if [[ "${HOST_PLATFORM}" != "${BUILD_PLATFORM}" ]] && [[ "${BUILD_WITH_CONDA_DEBUG:-0}" != 1 ]]; then
     EXTRA_CB_OPTIONS="${EXTRA_CB_OPTIONS:-} --no-test"
 fi
-{%- elif test == "native_and_emulated" -%}
+{%- elif test == "native_and_emulated" and conda_build_tool != "rattler-build" -%}
```

Comment:
is this correct? If we need `--no-test` then this is going to be missing in the rattler-build case, right?

Reply:
Initially it was disabled by me, but now that `--no-test` is working for rattler-build I need to remove this check

Reply:
I agree this looks wrong.

Reply:
I reverted these changes.